ui/components/Layout.tsx

```diff
@@ -160,6 +160,14 @@ function Layout({ className, children }: Props) {
                     className={n.sub && "sub-item"}
                   />
                 ))}
+
+                <StyleLinkTab
```

Comment:
I think you could add this to the navItems by giving the StyleLinkTab in the _.map something like `href={n.href}` and throwing `{label: "Docs", href: "https://docs.gitops.weave.works/"}` in that navItems array. Definitely not required but it follows the convention that's already there.